src/main/java/roomescape/controller/ReservationController.java

```diff
@@ -0,0 +1,56 @@
+package roomescape.controller;
+
+import jakarta.validation.Valid;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import roomescape.controller.dto.request.ReservationCreateRequest;
+import roomescape.controller.dto.response.ReservationResponse;
+import roomescape.domain.Reservation;
+
+import java.net.URI;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.NoSuchElementException;
+import java.util.concurrent.atomic.AtomicLong;
+
+@RestController
+@RequestMapping("/reservations")
+public class ReservationController {
+
+    private AtomicLong index = new AtomicLong(1);
+    private List<Reservation> reservations = new ArrayList<>();
+
+    @GetMapping
+    public ResponseEntity<List<ReservationResponse>> getReservations() {
+        List<ReservationResponse> responses = reservations.stream()
+                .map(ReservationResponse::from).toList();
```

Comment:
```suggestion
        List<ReservationResponse> responses = reservations.stream()
                .map(ReservationResponse::from)
                .toList();
```

리뷰하다보면 보통 저희 동아리는 `.` 단위로 개행을 해주더라고요